app/build.gradle

```diff
@@ -51,7 +57,7 @@ dependencies {
 
     // Dagger
     compile dagger
```

Comment:
`compile`, `testCompile`, etc. are deprecated now. Should switch to `implementation`, `testImplementation`, etc. https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation